include/openspace/util/distanceconversion.h

```diff
@@ -200,46 +154,76 @@ constexpr std::string_view nameForDistanceUnit(DistanceUnit unit, bool pluralFor
         case DistanceUnit::Mile:
         case DistanceUnit::League:
             if (pluralForm) {
-                return DistanceUnitNamesPlural[static_cast<int>(unit)];
+                return DistanceUnitNames[static_cast<int>(unit)].plural;
             }
             else {
-                return DistanceUnitNamesSingular[static_cast<int>(unit)];
+                return DistanceUnitNames[static_cast<int>(unit)].singular;
             }
         default:
             throw ghoul::MissingCaseException();
     }
 }
 
-constexpr DistanceUnit distanceUnitFromString(std::string_view unitName) {
-    int found = -1;
-    int i = 0;
-    for (std::string_view val : DistanceUnitNamesSingular) {
-        if (val == unitName) {
-            found = i;
-            break;
-        }
-        i++;
+constexpr std::string_view abbreviationForDistanceUnit(DistanceUnit unit) {
+    switch (unit) {
+        case DistanceUnit::Nanometer:
+        case DistanceUnit::Micrometer:
+        case DistanceUnit::Millimeter:
+        case DistanceUnit::Centimeter:
+        case DistanceUnit::Decimeter:
+        case DistanceUnit::Meter:
+        case DistanceUnit::Kilometer:
+        case DistanceUnit::AU:
+        case DistanceUnit::Lighthour:
+        case DistanceUnit::Lightday:
+        case DistanceUnit::Lightmonth:
+        case DistanceUnit::Lightyear:
+        case DistanceUnit::Parsec:
+        case DistanceUnit::Kiloparsec:
+        case DistanceUnit::Megaparsec:
+        case DistanceUnit::Gigaparsec:
+        case DistanceUnit::Gigalightyear:
+        case DistanceUnit::Thou:
+        case DistanceUnit::Inch:
+        case DistanceUnit::Foot:
+        case DistanceUnit::Yard:
+        case DistanceUnit::Chain:
+        case DistanceUnit::Furlong:
+        case DistanceUnit::Mile:
+        case DistanceUnit::League:
+            return DistanceUnitNames[static_cast<int>(unit)].abbreviation;
+        default:
+            throw ghoul::MissingCaseException();
```

Comment:
This looks like a strange switch? Why is it a switch and note something else when there is only one case?

Reply:
This is already how the similar function above was implemented so I just did it the exact same way 😅 

I believe the reason may be to be sure that we throw a meaningful exception if the unit for some reason does not exist among the list of units.  But yeah, I agree that it's a bit weird and it could probably be done some other way

Reply:
conclusion from discussion with Alex:  the switch could be removed here